samples/notebooks/prediction/SDK_Custom_Predict.ipynb

```diff
@@ -861,6 +861,8 @@
     "\n",
     "First authorize Google auth library to access the Cloud Platform with Google user credentials. This step requires to put user inputs. If you are using a non-interactive shells, you should open a terminal to run this command. If you are using Vertex Workbench, you will need to open [a terminal](https://jupyterlab.readthedocs.io/en/stable/user/terminal.html) to run this command.\n",
     "\n",
+    "Note that if you are running this command on machines without a window manager/browser, it will prompt a command `gcloud auth application-default login --remote-bootstrap=\"...\"` which requires you to run on machines where you can launch a web browser, for example, a laptop, and paste the result to this command to continue the login flow. If you are running this command on machines with web browsers, it should have just worked and there is no need to use a separate machine to do it.\n",
+    "\n",
     "This command will print the credential file as `Credentials saved to file: [CREDENTIALS_FILE]`."
    ]
   },
```

Comment:
Let's remove the `!gcloud auth application-default login` cell since we don't expect users to run it in the notebook.

Reply:
We still need to run this to generate the command to run outside of Vertex Workbench.

Reply:
But they'll have to run it in the terminal right? Not as a cell in the notebook.

Reply:
Depending on where the notebook is run. It's ok to run this in a cell if it's allowed to launch a browser.

In this case, do you think the above change "First authorize Google auth library to access the Cloud Platform with Google user credentials by running the following:" is still needed?

Reply:
Ok, then we can leave as a cell then.

That first part of my comment is not needed then. Please still apply the rest of the rewording suggestions in that comment though.

Reply:
Ack.